packages/packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api",
-  "version": "2.3.0",
+  "version": "2.3.1-alpha.0",
```

Comment:
should we just publish a new release pkg version as part of this pr so @Orta21 doesn't have to duplicate effort?

Reply:
Yea might as well